src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -25,3 +25,5 @@ public net.minecraft.client.KeyMapping clickCount
 # itemHandler
 public net.minecraft.world.entity.decoration.GlowItemFrame getFrameItemStack()Lnet/minecraft/world/item/ItemStack;
 public net.minecraft.world.entity.decoration.ItemFrame getFrameItemStack()Lnet/minecraft/world/item/ItemStack;
+
+public net.minecraft.world.level.block.state.BlockBehaviour hasCollision # hasCollision
```

Comment:
do we need this? i mean if it isnt avaiable through sth else

Reply:
it's the most efficient way to access it nicely.